enigmatry-entry-blueprint-app/package.json

```diff
@@ -51,9 +51,10 @@
     "@enigmatry/eslint-config": "16.0.0",
     "@enigmatry/scss-foundation": "16.0.0",
     "@enigmatry/stylelint-config": "16.0.0",
-    "@types/node": "^16.18.86",
     "@types/jest": "29.5.12",
+    "@types/node": "^16.18.86",
     "jest": "29.7.0",
+    "jest-junit": "^16.0.0",
```

Comment:
Why this library? Any reason not to use default format (lcov)?

Reply:
https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/publish-test-results-v2?view=azure-pipelines&tabs=trx%2Ctrxattachments%2Cyaml
The following formats are supported: CTest, JUnit, NUnit 2, NUnit 3, Visual Studio Test (TRX) and xUnit 2.